### ISSUE TITLE
Tests fail with Jenkins 1.6x - Diff is 718 characters long. Set self.maxDiff to None to see it.

### ISSUE THREAD

**Issue by hydrapolic** [salimfadhley/jenkinsapi#406]
Hello, trying to run tests with release 0.2.29 fails:

```
======================================================================
FAIL: test_get_job_and_update_config (jenkinsapi_tests.systests.test_jenkins.JobTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/test/jenkinsapi-0.2.29/jenkinsapi_tests/systests/test_jenkins.py", line 53, in test_get_job_and_update_config
    self.assertEquals(config.strip(), EMPTY_JOB.strip())
nose.proxy.AssertionError: '<?xml version="1.0" encoding="UTF-8"?><project>\n  <act[467 chars]ect>' != '<?xml version=\'1.0\' encoding=\'UTF-8\'?>\n<project>\n[486 chars]ect>'
Diff is 718 characters long. Set self.maxDiff to None to see it.
    """Fail immediately, with the given message."""
>>  raise self.failureException('\'<?xml version="1.0" encoding="UTF-8"?><project>\\n  <act[467 chars]ect>\' != \'<?xml version=\\\'1.0\\\' encoding=\\\'UTF-8\\\'?>\\n<project>\\n[486 chars]ect>\'\nDiff is 718 characters long. Set self.maxDiff to None to see it.')

-------------------- >> begin captured logging << --------------------
requests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost
requests.packages.urllib3.connectionpool: DEBUG: "GET /api/python HTTP/1.1" 200 285
requests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost
requests.packages.urllib3.connectionpool: DEBUG: "GET /api/python HTTP/1.1" 200 285
requests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost
requests.packages.urllib3.connectionpool: DEBUG: "GET /api/python?tree=jobs%5Bname%2Ccolor%2Curl%5D HTTP/1.1" 200 104
requests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost
requests.packages.urllib3.connectionpool: DEBUG: "GET /api/python HTTP/1.1" 200 285
requests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost
requests.packages.urllib3.connectionpool: DEBUG: "GET /job/create_sfgzwwimll/api/python HTTP/1.1" 200 397
requests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost
requests.packages.urllib3.connectionpool: DEBUG: "POST /job/create_sfgzwwimll/doDelete HTTP/1.1" 302 0
requests.packages.urllib3.connectionpool: DEBUG: "GET / HTTP/1.1" 200 3232
requests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost
requests.packages.urllib3.connectionpool: DEBUG: "GET /api/python HTTP/1.1" 200 252
requests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost
requests.packages.urllib3.connectionpool: DEBUG: "GET /api/python HTTP/1.1" 200 252
requests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost
requests.packages.urllib3.connectionpool: DEBUG: "GET /api/python HTTP/1.1" 200 252
requests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost
requests.packages.urllib3.connectionpool: DEBUG: "POST /createItem?name=config_yvctucivwm HTTP/1.1" 200 0
requests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost
requests.packages.urllib3.connectionpool: DEBUG: "GET /api/python HTTP/1.1" 200 288
requests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost
requests.packages.urllib3.connectionpool: DEBUG: "GET /job/config_yvctucivwm/api/python HTTP/1.1" 200 296
requests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost
requests.packages.urllib3.connectionpool: DEBUG: "GET /api/python HTTP/1.1" 200 288
requests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost
requests.packages.urllib3.connectionpool: DEBUG: "GET /api/python HTTP/1.1" 200 288
requests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost
requests.packages.urllib3.connectionpool: DEBUG: "GET /api/python?tree=jobs%5Bname%2Curl%2Ccolor%5D HTTP/1.1" 200 107
requests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost
requests.packages.urllib3.connectionpool: DEBUG: "GET /job/config_yvctucivwm/api/python HTTP/1.1" 200 296
requests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost
requests.packages.urllib3.connectionpool: DEBUG: "GET /job/config_yvctucivwm/config.xml HTTP/1.1" 200 511

Same I see here:
https://travis-ci.org/salimfadhley/jenkinsapi/jobs/93144744

Tried with Python 3.4 on Jenkins latest and LTS.
```

**Comment by lechat** [comment]
Yeah, I've seen that. Haven't got time to look at the cause yet.

**Comment by hydrapolic** [comment]
Works on Jenkins <1.6x.

**Comment by lechat** [comment]
The test failed because of the format of the job's xml - there were some more CRs on top and now they are removed. I already fixed that in fcefaaa.

**Comment by hydrapolic** [comment]
Great news, thanks!
